pkg/provider/aci.go

```diff
@@ -499,11 +499,11 @@ func (p *ACIProvider) CreatePod(ctx context.Context, pod *v1.Pod) error {
 		count = count + len(*container.Ports)
 	}
 	ports := make([]azaci.Port, 0, count)
-	for _, container := range *containers {
-		for _, containerPort := range *container.Ports {
-
+	for c := range *containers {
+		container := ((*containers)[c]).Ports
```

Comment:
The variable name `container` looks confusing. Can we rename it to `containerPorts`?

Reply:
done